docs/src/app/(public)/(content)/react/handbook/styling/page.mdx

```diff
@@ -0,0 +1,126 @@
+# Styling
+
+<Subtitle>A guide to styling Base UI components with your preferred styling engine.</Subtitle>
+<Meta
+  name="description"
+  content="Learn how to make the most of Base UI's accessibility features and guidelines."
+/>
+
+Base UI components are unstyled, don't bundle CSS, and are compatible with Tailwind, CSS Modules, CSS-in-JS, or any other styling solution you prefer. You retain total control of your styling layer.
+
+## Style hooks
+
+### CSS Classes
+
+Every component accepts a `className` prop which you can use to pass Tailwind classes or custom CSS classes to its DOM element. The `className` prop, in addition to accepting a string, can be defined as a function that takes a component state as an argument:
+
+```tsx title="switch.tsx"
+<Switch.Thumb className={(state) => (state.checked ? 'checked' : 'unchecked')} />
+```
+
+### Data attributes
+
+Data attributes are provided as style hooks. Component states can be styled with data attributes such as `[data-open]`. Certain props can be targeted with data attributes such as `[data-side]`. Some components provide `[data-starting-style]` and `[data-ending-style]` attributes for styling CSS transitions. Check out each component's API reference for a complete list of available data attributes.
+
+### CSS custom properties
+
+Some components expose CSS custom properties to aid with styling, such as `--accordion-panel-height`, `--transform-origin`, and `--active-tab-width`.
```

Comment:
- Totally a nit, but would feel better in the side nav if we put CSS classes and CSS vars closer together
- I'd call this section "CSS variables" since it's the colloquial name that most people would rather recognise

```suggestion
### CSS custom properties

Some components expose CSS custom properties to aid with styling, such as `--accordion-panel-height`, `--transform-origin`, or `--active-tab-width`.
```